resources/stats.py

```diff
@@ -0,0 +1,53 @@
+"""All the Statistics API endpoints"""
+
+from flask import current_app, make_response, jsonify
+from flask_restful import Resource
+from database.models import Quote, Author
+
+
+def get_query_most_popular(field, top_n):
+    """Returns Query to get top_n items with given field"""
+
+    return [
+        {"$project": {field: 1}},
+        {"$unwind": f"${field}"},
+        {"$group": {"_id": f"${field}", "count": {"$sum": 1}}},
+        {"$sort": {"count": -1}},
+        {"$limit": top_n},
+    ]
+
+
+class StatsApi(Resource):
+    """All the Statistics API endpoints"""
+
+    def get(self):
+        """Get Statistics for Quotes, Authors and Tags"""
+
+        try:
+            top_tags_query = get_query_most_popular("tags", 5)
+            tag_cursor = Quote.objects(author__exists=True).aggregate(top_tags_query)
+            current_app.logger.info("GET Stats - FETCHED TOP Tags")
+
+            top_author_query = get_query_most_popular("author", 1)
```

Comment:
Can we also get 
- total quotes by this author
- most popular tags associated with this author

Reply:
Done.